mephisto/client/cli.py

```diff
@@ -238,25 +295,38 @@ def get_help_arguments(args):
         if len(abstraction_equal_split) == 1:
             # querying about the general abstraction
             if abstraction == "blueprint":
-                click.echo(
-                    f"The blueprint determines the task content. Valid blueprints are {get_valid_blueprint_types()}"
+                click.echo("The blueprint determines the task content.\n")
+                valid_blueprints_text = """**Valid blueprints are:**"""
+                print_out_valid_options(
+                    valid_blueprints_text, get_valid_blueprint_types()
                 )
                 return
             elif abstraction == "architect":
                 click.echo(
-                    f"The architect determines the server where a task is hosted. Valid architects are {get_valid_architect_types()}"
+                    "The architect determines the server where a task is hosted.\n"
+                )
+                valid_architect_text = """**Valid architects are:**"""
+                print_out_valid_options(
+                    valid_architect_text, get_valid_architect_types()
                 )
                 return
             elif abstraction == "requester":
                 click.echo(
-                    f"The requester is an account for a crowd provider. Valid requester types are {get_valid_provider_types()}. \n"
-                    "Use `mephisto requesters` to see registered requesters, and `mephisto register <requester type>` to register."
+                    f"The requester is an account for a crowd provider. \n"
+                    "Use `mephisto requesters` to see registered requesters, and `mephisto register <requester type>` to register.\n"
+                )
+                valid_requester_text = """**Valid requesters are:**"""
+                print_out_valid_options(
+                    valid_requester_text, get_valid_provider_types()
                 )
                 return
             elif abstraction == "provider":
+                # TODO: Use markdown list here
```

Comment:
Do you have a plan for executing this TODO, or abandoning?

Reply:
I already executed it, the print_out_valid_options method uses the markdown list.

I forgot to remove the TODO though.